web/rootfs/defaults/settings-config.js

```diff
@@ -63,7 +63,8 @@
 {{ $ENABLE_LOCAL_RECORDING_SELF_START := .Env.ENABLE_LOCAL_RECORDING_SELF_START | default "false" | toBool -}}
 {{ $DISABLE_PROFILE := .Env.DISABLE_PROFILE | default "false" | toBool -}}
 {{ $ROOM_PASSWORD_DIGITS := .Env.ROOM_PASSWORD_DIGITS | default "false" -}}
-
+{{ $WHITEBOARD_COLLAB_SERVER_BASE_URL := .Env.WHITEBOARD_COLLAB_SERVER_BASE_URL | default "" -}}
```

Comment:
Can you please rename this to be `WHITEBOARD_COLLAB_SERVER_PUBLIC_URL` ? That would make it analogous to Etherpad, where use the "base" URL when it's builtin, and the other one when it's external.